### DISCUSSION TITLE
Pointer array assignment problem

### DISCUSSION THREAD

**Discussion by edanla** [protobuf-c/protobuf-c#621] [Q&A]
`struct Student{
       char *name;
       char *addr;
       int age;    
};

struct StudentList{
       size_t num;
       Student **stu;
};
`
This is the format I generated according to the protobuf.proto file. I want to pass the student information assignment now. How should I assign the allocation space?

**Comment by lipnitsk** [comment]
Please see example and test code to understand how protobuf-c works.
